CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [1.3.0] - 2017-07-24
 
 - Add `honeysql` at version 0.6.3
+- Update ring/ring-defaults to version 0.3.0
```

Comment:
Should also note that this version bump eliminates the need for the following exclusion carried around by many downstream projects in their `project.clj` files:

    [ring/ring-defaults nil :exclusions [javax.servlet/servlet-api]]